src/components/atoms/SfSearchBar/SfSearchBar.html

```diff
@@ -0,0 +1,13 @@
+<div class="sf-search-bar">
+    <input class="sf-search-bar__input" 
+           type="text" 
```

Comment:
hey @martaradziszewska , I think `type="search"` is better here :)

Reply:
search type adds x button, i'm worried it could look not like in designs

Reply:
yup, varies slightly by device, but it's an HTML5 standard and gain in usability.
ads the "search" button on mobile keyboard, which is best to use especially on mobile devices or browsers that don't show the "go" button for text input...
anyway, if you think it's most important to follow the designs exactly, I suggest adding a `type` prop, with text as default.